owncloud/phoenix
# Support translations of nav items and file actions

## Description
Menu items shall be translated as well.

The proposed solution is far from perfect - but fits the needs.

## How Has This Been Tested?
- switch browser to German
- open the menu
- see the items translated

## Screenshots (if appropriate):

## Types of changes


## Checklist:



## Open tasks:
